Doom-Mummies/Welcome-to-Nashville
# Mj ordered list results

# Description

turned results for restaurants into an ordered list and improved code for results

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1. ``git fetch --all``
2.  `` git checkout mj-orderedListResults``
3. verify that restaurant results show up as an ordered list and is limited to 4


# Checklist:
